examples/terraform/aws/output.tf

```diff
@@ -30,7 +30,7 @@ output "kubeone_hosts" {
       ssh_agent_socket     = "${var.ssh_agent_socket}"
       ssh_port             = "${var.ssh_port}"
       ssh_private_key_file = "${var.ssh_private_key_file}"
-      ssh_user             = "ubuntu"
+      ssh_user             = "${var.ssh_username}"
```

Comment:
Why the variable name is not the same as in the output?

Reply:
for historical reasons, `ssh_user` is kubeone's API, `$var.ssh_username` is for consistency (GCE example already had this).